src/WCSimPMTQE.cc

```diff
@@ -38,7 +38,7 @@ G4double WCSimDetectorConstruction::GetPMTQE(G4String CollectionName, G4double P
 
   // return 0 for wavelenght outside the range
   if (flag==1){
-    if (PhotonWavelength <= low_wl || PhotonWavelength >= high_wl || PhotonWavelength <=280 || PhotonWavelength >=660){
+    if (PhotonWavelength <= low_wl || PhotonWavelength >= high_wl ){
```

Comment:
I guess this is an optimisation for ID PMTs but the OD will use WLS plates that can detect photons with less than 280nm wavelength. I will revert but this will need to be included for the OD to function correctly.

Reply:
That's right. But anyway, why is the test failing? If extending the possible wavelengths changes the results for the ID, doesn't that mean that they *should* be extended, because some photons are being missed?